v2/cli/term/event_page.go

```diff
@@ -93,29 +102,47 @@ func (e *eventPage) refresh(eventID string) {
 }
 
 func (e *eventPage) fillEventInfo(event core.Event) {
+
 	e.eventInfo.Clear()
-	e.eventInfo.SetText(
-		fmt.Sprintf(
-			"[yellow]Event: [white]%s\n"+
-				"[yellow]Source: [white]%s\n"+
-				"[yellow]Type: [white]%s\n"+
-				"[yellow]Time Created: [white]%s",
-			event.ID,
-			event.Source,
-			event.Type,
-			formatDateTimeToString(*event.Created),
-		),
+	e.eventInfo.SetTitle(event.ID)
+	eventText := fmt.Sprintf(
+		"[yellow]Source: [white]%s\n"+
+			"[yellow]Type: [white]%s\n"+
+			"[yellow]Time Created: [white]%s",
+		event.Source,
+		event.Type,
+		formatDateTimeToString(*event.Created),
 	)
 
+	// Add qualifiers (if any) to event info box
+	for k, v := range event.Qualifiers {
+		eventText = eventText + fmt.Sprintf("\n[yellow]%s: [white]%s", k, v)
+	}
+
+	// Add labels (if any) to event info box
+	for k, v := range event.Labels {
+		eventText = eventText + fmt.Sprintf("\n[yellow]%s: [white]%s", k, v)
+	}
+
+	e.eventInfo.SetText(eventText)
```

Comment:
We need to differentiate between qualifiers and labels somehow.

Can we conditionally add "Qualifiers:" if there are any, then indent each of the qualifier lines?

Same for labels. Conditionally add "Labels:" if there are any, then indent each of the label lines.